vm-setup/roles/v1aX_integration_test/tasks/node_reuse_kcp.yml

```diff
@@ -0,0 +1,200 @@
+---
+  - set_fact:
+      NUMBER_OF_BMH_KCP: "{{ NUM_OF_MASTER_REPLICAS|int }}"
+      NUMBER_OF_BMH_MD: "{{ NUM_OF_WORKER_REPLICAS|int }}"
+      NUMBER_OF_ALL_BMH: "{{ NUM_OF_WORKER_REPLICAS|int + NUM_OF_MASTER_REPLICAS|int }}"
+  
+  - name: Scale worker down to 0.
```

Comment:
You could use [community.kubernetes.k8s_scale ](https://docs.ansible.com/ansible/latest/collections/community/kubernetes/k8s_scale_module.html)to accomplish this:
```
- name: Scale worker down to 0
  community.kubernetes.k8s_scale:
    api_version: MD_API
    kind: MachineDeployment
    name: "{{ CLUSTER_NAME }}"
    namespace: "{{ NAMESPACE }}"
    replicas: 0
```

Reply:
seems this is not doable:
` "Scale request failed: 415\nReason: Unsupported Media Type\nHTTP response headers: HTTPHeaderDict({'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': '4a0678b4-f217-410f-8429-e0c10839c4ad', 'X-Kubernetes-Pf-Prioritylevel-Uid': '69a02ea6-db61-48cd-8ff1-9312121b0297', 'Date': 'Thu, 20 May 2021 18:29:03 GMT', 'Content-Length': '293'})\nHTTP response body: b'{\"kind\":\"Status\",\"apiVersion\":\"v1\",\"metadata\":{},\"status\":\"Failure\",\"message\":\"the body of the request was in an unknown format - accepted media types include: application/json-patch+json, application/merge-patch+json, application/apply-patch+yaml\",\"reason\":\"UnsupportedMediaType\",\"code\":415}\\n'\nOriginal traceback: \n  File \"/usr/local/lib/python3.8/dist-packages/openshift/dynamic/client.py\", line 42, in inner\n    resp = func(self, *args, **kwargs)\n\n  File \"/usr/local/lib/python3.8/dist-packages/openshift/dynamic/client.py\", line 235, in request\n    return self.client.call_api(\n\n  File \"/usr/local/lib/python3.8/dist-packages/kubernetes/client/api_client.py\", line 340, in call_api\n    return `

Reply:
seems this is not possible to use:
` "Scale request failed: 415\nReason: Unsupported Media Type\nHTTP response headers: HTTPHeaderDict({'Cache-Control': 'no-cache, private', 'Content-Type': 'application/json', 'X-Kubernetes-Pf-Flowschema-Uid': '4a0678b4-f217-410f-8429-e0c10839c4ad', 'X-Kubernetes-Pf-Prioritylevel-Uid': '69a02ea6-db61-48cd-8ff1-9312121b0297', 'Date': 'Thu, 20 May 2021 18:29:03 GMT', 'Content-Length': '293'})\nHTTP response body: b'{\"kind\":\"Status\",\"apiVersion\":\"v1\",\"metadata\":{},\"status\":\"Failure\",\"message\":\"the body of the request was in an unknown format - accepted media types include: application/json-patch+json, application/merge-patch+json, application/apply-patch+yaml\",\"reason\":\"UnsupportedMediaType\",\"code\":415}\\n'\nOriginal traceback: \n  File \"/usr/local/lib/python3.8/dist-packages/openshift/dynamic/client.py\", line 42, in inner\n    resp = func(self, *args, **kwargs)\n\n  File \"/usr/local/lib/python3.8/dist-packages/openshift/dynamic/client.py\", line 235, in request\n    return self.client.call_api(\n\n  File \"/usr/local/lib/python3.8/dist-packages/kubernetes/client/api_client.py\", line 340, in call_api\n    return `